streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -516,6 +524,12 @@ private void completeAllProcessableWork() {
                 task.commit();
                 captureOutputsAndReEnqueueInternalResults();
             }
+            if (task.hasRecordsQueued()) {
+                log.info("Due to the {} configuration, there are currently some records that can't be processed." +
+                             " Advancing wall-clock time or enqueuing records on the empty partitions will allow" +
```

Comment:
`empty partitions` -> should we say `empty topics` ?

Just to clarify that some partitions from _different_ topics are empty?